resources/js/Components/Navbar.vue

```diff
@@ -0,0 +1,112 @@
+<template>
```

Comment:
Can you go into more detail for what this functionality is for? It doesn't seem to relate specifically to the Booking feature. Will this cause conflicts with the navbar/theme tasks?

Reply:
This was just used to test the dynamic tables that I created. Forgot to clean that up but will remove it now.